mysql-ballerina/client.bal

```diff
@@ -132,7 +132,7 @@ public client class Client {
 # + user - Username for the database connection
 # + password - Password for the database connection
 # + database - Name of the database
-# + options - Mysql datasource `Options` to be configured
+# + options - Mysql datasource `mysql:Options` to be configured
 # + connectionPool - Properties for the connection pool configuration. Refer `sql:ConnectionPool` for more details
```

Comment:
```suggestion
# + connectionPool - Properties for the connection pool configuration. For more details, see the `sql:ConnectionPool`.
```
Should we give the link here?

Reply:
IMO -1, link is attached in Module.md file. No need in here.